dirs/dirs.go

```diff
@@ -165,8 +169,13 @@ var (
 )
 
 type SnapDirOptions struct {
-	// HiddenSnapDataDir determines if the snaps' data is in ~/.snap/data instead of ~/snap
+	// HiddenSnapDataDir determines if the snaps' data is in ~/.snap/data instead
+	// of ~/snap
 	HiddenSnapDataDir bool
+
+	// MigratedToExposedHome determines if the snap's directory in ~/Snap has been
+	// initialized.
```

Comment:
Did you mean "migrated" here?

Reply:
From the code below it looks like "initialized" is deliberate here, maybe expand the comment a bit about what "initialized" means.

Reply:
No, I thought "initialized" might be clearer in this case. For this part of the migration, it's not a directory moving from one place to another, ~/Snap is being created and initialized with the contents from the revisioned directory.

Reply:
Done